docs/projects/version-control/config-as-code-reference.md

```diff
@@ -213,17 +214,36 @@ connectivity_policy {
 versioning_strategy {
 
     donor_package {
-        step = "Deploy a Package"
+        step = "deploy-a-package"
     }
 }
 ```
 
+## Slugs in OCL
+
+Previously, Git projects would reference shared resources using their name. This had a side-effect causing API responses for Git projects to contain names instead of IDs.
```

Comment:
How previously? How recently. Add some versions here, even if we need to update it just after it goes out.
`Prior to version 2022.3.x`